crates/tabby/src/services/code.rs

```diff
@@ -185,6 +185,14 @@ fn create_hit(scores: CodeSearchScores, doc: TantivyDocument) -> CodeSearchHit {
             code::fields::CHUNK_GIT_URL,
         )
         .to_owned(),
+        // commit is introduced in v0.23, but it is also a required field
+        // so we need to handle the case where it's not present
+        commit: get_json_text_field_or_default(
```

Comment:
we can mark `commit` field as `optional` by now, and flip it to required after two release.